cmd/generate/config/rules/settlemint.go

```diff
@@ -0,0 +1,70 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func SettlemintPersonalAccessToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Found a Settlemint Personal Access Token.",
+		RuleID:      "settlemint-personal-access-token",
+		Regex:       utils.GenerateUniqueTokenRegex(`(sm_pat_)[a-zA-Z0-9]{12,}`, false),
```

Comment:
@Kemosabert in an earlier comment you said secrets have 12 characters after `sm_pat_`. Only 12? If so can we limit the `[a-zA-Z0-9]` range to `{12}` instead of `{12,}`

Reply:
I meant 12 characters after the prefix, sorry for the confusion. I'll adjust the regexes 👍